.circleci/config.yml

```diff
@@ -36,27 +58,19 @@ workflows:
                 - maintenance
             exclude:
               - os: windows
-                node_version: lts
+                node_version: latest
               - os: windows
                 node_version: maintenance
-      - release-management/release-package:
-          sign: true
-          github-release: true
+      - release-management/test-nut:
+          name: nuts-on-linux
+          sfdx_version: latest
           requires:
             - release-management/test-package
-          filters:
-            branches:
-              only: main
-        release-management/test-nut:
-          matrix:
-            parameters:
-              os:
-                - linux
-                - windows
-              sfdx_version:
-                - latest
-              node_version:
-                - latest
+      - release-management/test-nut:
+          name: nuts-on-windows
+          node_version: lts
+          sfdx_version: latest
+          os: windows
           requires:
             - release-management/test-package
```

Comment:
run the windows nut AFTER the linux once.  Sometimes they collide and error out.

Reply:
Also, we can start this repo off on scheduledPipelines, right?